ratslam_lv:
  ros__parameters:
    image_crop_x_min: 40
    image_crop_x_max: 600
    image_crop_y_min: 150
    image_crop_y_max: 300
    template_x_size: 60
    template_y_size: 20
    vt_match_threshold: 0.073
    vt_panoramic: 0
    vt_patch_normalise: 2
    vt_shift_match: 25
    vt_step_match: 5
    vt_normalisation: 0.0
    # draw
    enable: true
    vt_window_width: 416
    vt_window_height: 480

ratslam_pc:
  ros__parameters:
    pc_cell_x_size: 1.0
    pc_dim_xy: 18
    exp_delta_pc_threshold: 2.0
    pc_vt_inject_energy: 0.2
    vt_active_decay: 1.0
    pc_vt_restore: 0.05
    # draw
    enable: true
    posecells_size: 250

ratslam_em:
  ros__parameters:
    exp_loops: 50
    exp_initial_em_deg: 180.0
    exp_correction: 0.5
    # draw
    enable: true
    exp_map_size: 500